src/ImuSensor.cc

```diff
@@ -116,6 +116,16 @@ bool ImuSensor::Load(const sdf::Sensor &_sdf)
     return false;
   }
 
+  // Set orientation reference frame
+  // TODO(adityapande-1995) : Add support for named frames like ENU using ign-gazebo
+  if (_sdf.ImuSensor()->Localization() == "CUSTOM") {
+    if (_sdf.ImuSensor()->CustomRpyParentFrame() == "") {
+      this->SetOrientationReference(ignition::math::Quaterniond(_sdf.ImuSensor()->CustomRpy()));
```

Comment:
`make codecheck` complains about this line and line 120 having a length over 80 characters. can you shorten or split over two lines?

Reply:
Done !